src/raftstore/store/util.rs

```diff
@@ -256,9 +256,10 @@ pub fn check_region_epoch(
             | AdminCmdType::InvalidAdmin
             | AdminCmdType::ComputeHash
             | AdminCmdType::VerifyHash => {}
-            AdminCmdType::Split | AdminCmdType::BatchSplit => check_ver = true,
             AdminCmdType::ChangePeer => check_conf_ver = true,
-            AdminCmdType::PrepareMerge
+            AdminCmdType::Split
```

Comment:
Any integration tests?

Reply:
No. Because it's hard to reproduce the case that **after raftstore sends an AskSplit to PD with conf version X, PD accepts it and send back a split command to raftstore, and btween these raftstore applied a conf change**.

Reply:
You can use failpoint to achieve the goal.